PsyshockPhysics/Physics/Authoring/ConvexColliderSmartBlobberSystem.cs

```diff
@@ -137,7 +137,7 @@ protected override void OnUpdate()
                 }
             }).Schedule();
 
-            Entities.ForEach((ref SmartBlobberResult result, in ConvexColliderBlobBakeData data) =>
+            Entities.WithReadOnly(hashmap).ForEach((ref SmartBlobberResult result, in ConvexColliderBlobBakeData data) =>
```

Comment:
Great find! This unfortunately doesn't have coverage in any of my test projects currently. That's something I hope a future project can fix.